### PR TITLE
Updates the contribution guidelines.

### PR DESCRIPTION
**Pull request by jessex**
## Description of the change

Updates the contribution guidelines: updates some outdated links, removes some unnecessary detail now formalized elsewhere, tweaks some language.

## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Configuration change (adjusts configuration to achieve some end, related to functionality, development, performance, or security)

## Related issues

> Closes [#XXXX]

### Development

These boxes should be checked by the submitter prior to merging:

- [x] Lint rules pass locally
- [x] Manual testing against realistic data has been performed locally

### Code review

These boxes should be checked by reviewers prior to merging:

- [ ] This pull request has a descriptive title and information useful to a reviewer
- [ ] This pull request has been moved out of a Draft state, has no "Work In Progress" label, and has assigned reviewers
- [ ] Potential security implications or infrastructural changes have been considered, if relevant

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -81,16 +79,18 @@ assign an issue to yourself if you intend to submit a pull request soon. If you
...
+into an area where you feel less comfortable or need to learn something new.
 
 ### Doing the Work
 
 **tl;dr:**
 1. Work on a feature branch
-1. [Run linting and tests often](README.md#linting--running-tests)
+1. Test your change manually by running the app locally
+1. [Run linting and tests often](README.md)
 1. Ensure your change has test coverage (_ideally complete coverage_)
```
Comment (jjdressel11) [inline CONTRIBUTING.md]: We don't have any tests for the dashboard yet - should this still be here?

Comment (jessex) [reply CONTRIBUTING.md]: I do want to leave it here in the guidelines, since it's an aspiration that pushes us towards bring tests back into the updated app structure. Any concern?

Comment (jjdressel11) [reply CONTRIBUTING.md]: Okay fair enough
